.testing.pylintrc

```diff
@@ -60,7 +60,7 @@ extension-pkg-whitelist=
 
 # Fileperms Lint Plugin Settings
 fileperms-default=0644
-fileperms-ignore-paths=setup.py,tests/runtests.py,tests/jenkins*.py,tests/saltsh.py,tests/buildpackage.py
+fileperms-ignore-paths=setup.py,tests/runtests.py,tests/jenkins*.py,tests/saltsh.py,tests/buildpackage.py,tests/unit/files/rosters/ansible/roster.py
```

Comment:
Why does this last file have wrong perms? Could we just fix the perms instead of ignoring it?

Reply:
Also, if you pulled in the latest master branch changes `.testing.pylintrc` is now gone.

Reply:
It's part of the kitchen tutorial, so as far as I'm aware it has to have that permissions.

Reply:
ok. the `.testing.pylintrc` issue is still valid though